crates/astria-composer/src/composer.rs

```diff
@@ -140,6 +157,13 @@ impl Composer {
         self.api_server.local_addr()
     }
 
+    /// Returns the socker address the grpc collector is served over
+    /// # Errors
+    /// Returns an error if the listener is not bound
+    pub fn grpc_collector_local_addr(&self) -> io::Result<SocketAddr> {
```

Comment:
```suggestion
    pub fn grpc_local_addr(&self) -> io::Result<SocketAddr> {
```
This is after all not the address of the collector, but the address of the entire server that is also serving the collector service.